symengine/lib/symengine_wrapper.pyx

```diff
@@ -2160,6 +2166,14 @@ cdef class MutableDenseMatrix(DenseMatrixBase):
         for k in range(0, self.cols):
             self[i, k], self[j, k] = self[j, k], self[i, k]
 
+    def applyfunc(self, f):
```

Comment:
This one uses the python defined method `_set`. The implementation above directly uses C++ method.